internal/commands/database/create.go

```diff
@@ -0,0 +1,133 @@
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/ui"
+	"github.com/UpCloudLtd/upcloud-go-api/v8/upcloud"
+	"github.com/UpCloudLtd/upcloud-go-api/v8/upcloud/request"
+	"github.com/spf13/pflag"
+)
+
+type createCommand struct {
+	*commands.BaseCommand
+	params createParams
+	wait   config.OptionalBoolean
+}
+
+func CreateCommand() commands.Command {
+	return &createCommand{
+		BaseCommand: commands.New("create", "Create a new database", "upctl database create"),
+	}
+}
+
+var defaultCreateParams = createParams{
+	CreateManagedDatabaseRequest: request.CreateManagedDatabaseRequest{
+		Plan: "2x2xCPU-4GB-100GB",
+		Type: upcloud.ManagedDatabaseServiceTypeMySQL,
+	},
+}
+
+type createParams struct {
+	request.CreateManagedDatabaseRequest
+	labels              []string
+	networks            []string
+	terminateProtection bool
+	dbType              string
+	properties          []string
+}
+
+func (s *createParams) processParams() error {
+	if len(s.labels) > 0 {
+		labelSlice, err := labels.StringsToSliceOfLabels(s.labels)
+		if err != nil {
+			return err
+		}
+		s.Labels = labelSlice
+	}
+
+	if s.terminateProtection {
+		s.TerminationProtection = &s.terminateProtection
+	}
+
+	if s.dbType != "" {
+		s.Type = upcloud.ManagedDatabaseServiceType(s.dbType)
+	}
+
+	if len(s.properties) > 0 {
+		var props request.ManagedDatabasePropertiesRequest
+		for _, prop := range s.properties {
+			parts := strings.SplitN(prop, "=", 2)
+			if len(parts) != 2 {
+				return fmt.Errorf("invalid property format: %s, expected key=value", prop)
+			}
+			key := upcloud.ManagedDatabasePropertyKey(parts[0])
+			props.Set(key, parts[1])
+		}
+
+		s.Properties = props
+	}
+
+	return nil
+}
+
+// InitCommand implements commands.InitializeCommand
+func (s *createCommand) InitCommand() {
+	flags := &pflag.FlagSet{}
+	s.params = createParams{CreateManagedDatabaseRequest: request.CreateManagedDatabaseRequest{}}
+	def := defaultCreateParams
+	flags.StringVar(&s.params.HostNamePrefix, "host-name-prefix", def.HostNamePrefix, "A host name prefix for the database")
+	flags.StringVar(&s.params.Title, "title", def.Title, "A short, informational description.")
+	flags.StringVar(&s.params.Plan, "plan", def.Plan, "Plan to use for the database. Run `upctl database plans [database type]` to list all available plans.")
+	flags.StringVar(&s.params.Zone, "zone", def.Zone, namedargs.ZoneDescription("database"))
+	flags.StringVar(&s.params.dbType, "type", string(def.Type), "Type of the database")
+	flags.StringSliceVar(&s.params.labels, "labels", def.labels, "Labels to describe the database in `key=value` format, multiple can be declared.\nUsage: --label env=dev\n\n--label owner=operations")
+	flags.StringSliceVar(&s.params.networks, "networks", def.networks, "A network interface for the database, multiple can be declared.\nUsage: --network family=IPv4,type=public\n\n--network type=private,network=037a530b-533e-4cef-b6ad-6af8094bb2bc,ip-address=10.0.0.1")
```

Comment:
I think we use singular for these parameters that can be defined multiple times 🤔 

```suggestion
	flags.StringSliceVar(&s.params.labels, "label", def.labels, "Labels to describe the database in `key=value` format, multiple can be declared.\nUsage: --label env=dev\n\n--label owner=operations")
	flags.StringSliceVar(&s.params.networks, "network", def.networks, "A network interface for the database, multiple can be declared.\nUsage: --network family=IPv4,type=public\n\n--network type=private,network=037a530b-533e-4cef-b6ad-6af8094bb2bc,ip-address=10.0.0.1")
```

Also, attaching network to DB is slightly different than with servers, but no need to update the example before adding parsing for the parameter. We could use similar way to parse the argument value. Another more recent example of the approach would be in `--node-group` argument of `kubernetes create` command.